### ISSUE TITLE
Cancel/Close Form - Option to unrelate form from workflow documents

### ISSUE THREAD

**Issue by dgrumieaux** [tsgrp/hpi#994]
When a form is no longer relevant for a workflow document, currently a user should 1) unrelate the workflow document and then 2) close or cancel the form. In some client scenarios, users are not aware of both of these steps and may forget to perform step 1. It would be nice to have a checkbox show up whenever a user is Canceling or Closing a form to give them the option to unrelate the documents then and there instead of having to manually perform step 1.

As far as design goes:

- This option should only show up if there are 1 or more workflow documents attached to the form at the time of Cancel/Close
- The option should default to checked (yes, remove relations) but be configurable from the HPI admin

@benallenallen @gsteimer thoughts?

**Comment by gsteimer** [comment]
Yeah, I agree.  It's easy to forget step 1

**Comment by dgrumieaux** [comment]
If we can't think of a good case where we'd want those relationships to stick around, we can just make this enhancement in the Cancel/Close form OC action executers and not even worry about HPI frontend enhancements.

**Comment by benallenallen** [comment]
There is a big difference between close and cancel though... i think it should only happen on CANCEL. Close is usually used once the form is approved, and we don't want to open up a can of worms of what happens with the freeze and not be able to get back to the doc that was attached when the form was closed/lose the history of what doc was attached.

**Comment by gsteimer** [comment]
Ha - @benallenallen I was just typing up the same thing...

**Comment by dgrumieaux** [comment]
Yeah, you're exactly right @benallenallen ... sounds like we want this to automatically happen for Cancel but never for Close.

**Comment by dgrumieaux** [comment]
I'll edit title for accuracy
